### PR TITLE
Razorpay sdk

### PR DESCRIPTION
**Pull request by mayankamencherla**
Integrates latest razorpay-php sdk API with the woocommerce razorpay plug in.

### CODE REVIEW

**Comment by captn3m0** [comment]
@mayankamencherla The logo commit is somehow included in this PR. You can rebase and drop it from this branch

File: razorpay-payments.php [hunk 1]
```diff
@@ -290,79 +295,55 @@ function check_razorpay_response()
...
-                            }
-                            else
-                            {
-                                $error = "RAZORPAY_ERROR: Invalid Response <br/>".$result;
-                            }
+                            $error = "RAZORPAY_ERROR: Invalid Response <br/>";
                         }
                     }
                 }
+                catch(Requests_Exception $e){
```
Comment (captn3m0) [inline razorpay-payments.php]: We won't be getting a Requests exception now. Razorpay PHP SDK has it's own exceptions.

File: razorpay-sdk/Razorpay.php [hunk 2]
```diff
@@ -0,0 +1,41 @@
+<?php
+
+// Include Requests
+/*require_once __DIR__.'/libs/Requests-1.6.1/library/Requests.php';
```
Comment (captn3m0) [inline razorpay-sdk/Razorpay.php]: We'll need to require this if the Requests class is not defined.

Comment (mayankamencherla) [inline razorpay-sdk/Razorpay.php]: But when I require Requests.php it runs into an error.

Comment (captn3m0) [inline razorpay-sdk/Razorpay.php]: *if the class is not defined*, which is the case in wordpress < 4.6

**Comment by mayankamencherla** [comment]
fixed up , @captn3m0 you can review.

**Comment by captn3m0** [comment]
Delete the `Requests/lib/{docs|examples|tests}` directories.

**Comment by mayankamencherla** [comment]
Done. @captn3m0

**Pull request merged**